module/zfs/qat_compress.c

```diff
@@ -36,7 +36,7 @@
  * Max instances in QAT device, each instance is a channel to submit
  * jobs to QAT hardware
  */
-#define	MAX_INSTANCES		6
+#define	MAX_INSTANCES		48
```

Comment:
This simply results in slightly larger arrays being allocated which are unused on the previous chipset, correct?  It would be helpful to update the comment to document the maximum values per-chipset.

Reply:
Yes, it is the maximum instances supported by the chipset. The real number of instances are defined in the QAT configure file. I will update comment, thanks!